randovania/games/prime1/logic_database/Tallon Overworld.txt

```diff
@@ -73,7 +73,10 @@ Extra - aabb: [-430.3307, 289.15143, -43.67576, -295.17084, 424.52155, 50.736755
   > Door to Waterfall Cavern
       Trivial
   > Pickup (Missile Expansion)
-      Morph Ball
+      Any of the following:
+          Morph Ball
+          # Single Room OoB https://www.youtube.com/watch?v=gY0SRbHkTqs
+          Space Jump Boots and Combat/Scan Dash (Intermediate) and Single Room Out of Bounds (Advanced) and Standable Terrain (Intermediate)
```

Comment:
Rewrite to add Slope Jump and Wall Boost and scan visor requirement

```All of the following:```
```Space Jump Boots and Single Room Out of Bounds (Advanced) and Standable Terrain (Intermediate)```
    ```Any of the following:```
    ```Scan Visor and Combat/Scan Dash (Intermediate)```
   ```Slope Jump (Advanced)```
   ```Morph Ball and Boost Ball and Wallboost (Hypermode)```

Reply:
Bro think about the wall boost suggestion for just a moment.

Reply:
Er, right. Forgot it isn't actually a wallboost. Feel free to use movement logic for that if you think it's more fitting and don't forget ```and not dock rando```.

Reply:
Sam...